meta-lmp-base/recipes-connectivity/bluez5/bluez5_5.65.bbappend

```diff
@@ -0,0 +1,15 @@
+# This enables using BSD library libedit for client/mesh features
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+PACKAGECONFIG:append = " client "
+
+PACKAGECONFIG[readline] = "--with-readline=readline,,readline"
+PACKAGECONFIG[libedit] = "--with-readline=libedit,,libedit"
```

Comment:
Something like that can help to raise a error when `readline` and `libedit`  is enabled on `PACKAGECONFIG`
```
PACKAGECONFIG[readline] = "--with-readline=readline,,readline,,,libedit"
PACKAGECONFIG[libedit] = "--with-readline=libedit,,libedit,,,readline"
```

Reply:
@quaresmajose Good catch, thanks! I'll add the conflict features.